vignettes/developer-guidelines.Rmd

```diff
@@ -0,0 +1,21 @@
+---
+title: "Developer Guidelines"
+date: "`r Sys.Date()`"
+vignette: >
+  %\VignetteIndexEntry{Developer Guidelines} %\VignetteEngine{knitr::rmarkdown}
+  %\usepackage[utf8]{inputenc}
+---
+
+## CRAN Release Checklist
+
+When deploying updates to CRAN, here a few updates that you need to check off before doing so:
+
+- Update version number in DESCRIPTION in format major.minor.patch (e.g. 1.2.1)
+- Update documentation with `devtools::document()`
```

Comment:
Does `devtools::document()` handle both building `README.md` from `README.Rmd` and rebuilding the vignettes? I'd been knitting the README manually and calling `pkgdown::build_site()` for the vignettes.

Reply:
No, it doesn't -- it only does package documentation. I've specified that and added provisions to re-knit the readme and call pkgdown to the list (and did it for this commit too!).